lib/generators/locomotive/install/templates/carrierwave_aws.rb

```diff
@@ -22,6 +22,13 @@
       region:             ENV['S3_BUCKET_REGION']
     }
 
+    # Use a different endpoint (eg: another provider such as Exoscale)
+    if ENV['S3_ENDPOINT'].present?
+      config.aws_credentials.config: AWS.config({
```

Comment:
I'm brand new to Ruby: shall I use dict/hash like syntax instead of dotted path?